workspaces/marketplace/plugins/marketplace/src/hooks/useQueryArrayFilter.ts

```diff
@@ -25,13 +25,9 @@ const evaluateParams = (
   newParams: URLSearchParams,
   filterName: string,
 ) => {
-  if (Array.isArray(newValues)) {
-    newValues.forEach(v => {
-      newParams.append(filterSearchParam, `${filterName}=${v}`);
-    });
-  } else {
-    newParams.append(filterSearchParam, `${filterName}=${newValues}`);
-  }
+  newValues.forEach(v => {
+    newParams.append(filterSearchParam, `${filterName}=${v}`);
+  });
```

Comment:
Is this change really correct -- and needed?

The `evaluateParams` signature says it can be called with a `string` or `string[]`?

Reply:
It's `(string | number)[]`, you're right... :+1: